peg/Parser.hx

```diff
@@ -352,7 +413,12 @@ class Parser {
 					return;
 				//$argName
 				case T_VARIABLE:
-					fn.addArg(parseVar(ctx, token.value));
+					var v = parseVar(ctx, token.value);
+					var t = parseDocTagParamType(fn, v.name);
+					if (t != null && v.type == TMixed) {
```

Comment:
Better check `v.type` before executing `parseDocTagParamType`

Reply:
done in 09f1f4c